site/src/patterns/card.md

```diff
@@ -48,28 +48,86 @@ This card type contains a super title labelled by the card's type, a title, an i
 </div>
 ```
 
-## Resource card
+## Icon card
 
-This card pattern makes use of the outer `.card` class and includes the use of the [`.media`](../components/media.md) and [`.image`](../components/image.md) components inside it. Also, because there is only one clickable element inside, it uses the [`.stretched-link` component](../components/stretched-link.md)
+This card pattern makes use of the outer `.card` class and includes the use of the [`.media`](../components/media.md) and [`.image`](../components/image.md) components inside it. Also, because there is only one clickable element inside, it uses the [`.stretched-link` component](../components/stretched-link.md). There are also several flexbox based atomics applied inside `.media` to ensure links justify at the ends of cards when placed in a grid.
 
 ```html
 <article class="card position-relative">
+	<div class="card-content display-flex">
+		<div class="media align-items-stretch">
+			<div class="media-left">
+				<div class="image image-64x64">
+					<img
+						alt="Write alt text here or if image is presentational only add aria-hidden attribute"
+						src="~/src/scaffold/media/video-media.png"
+					/>
+				</div>
+			</div>
+			<div class="media-content display-flex flex-direction-column align-items-space-between">
```

Comment:
It doesn't seem like we have `space-between` in our `align-items` atomics map.

Reply:
Whoops, removed this line. space-between is not a valid value for align-items